examples/pytorch/rgcn-hetero-ogbn-mag/model.py

```diff
@@ -322,7 +322,8 @@ def inference(
                     h = self._apply_layers(i, h)
 
                 for ntype in h:
-                    y[ntype][out_nodes[ntype]] = h[ntype]
+                    if ntype in out_nodes:
```

Comment:
Is it equivalent to replace L324-325 with `for ntype in out_nodes`?

Reply:
Yeah we can do so, I modified it.